src/common/tests/CommonUtilsUT.cpp

```diff
@@ -1664,4 +1664,28 @@ TEST_F(CommonUtilsTest, CheckLockoutForFailedPasswordAttempts)
         EXPECT_NE(0, CheckLockoutForFailedPasswordAttempts(m_path, nullptr));
         EXPECT_TRUE(Cleanup(m_path));
     }
+}
+
+TEST_F(CommonUtilsTest, CleanDoubleBackslashes)
+{
+    const char* expected = "\nThis is a test\n\nHere is another line\nAnd another\n\n\nEnd\n";
+    char* value = NULL;
+
+    EXPECT_EQ(nullptr, CleanDoubleBackslashes(nullptr));
+    EXPECT_EQ(nullptr, CleanDoubleBackslashes(""));
+        
+    EXPECT_STREQ(expected, value = CleanDoubleBackslashes("\\nThis is a test\\n\\nHere is another line\\nAnd another\\n\\n\\nEnd\\n"));
+    FREE_MEMORY(value);
+
+    EXPECT_STREQ(expected, value = CleanDoubleBackslashes("\nThis is a test\n\nHere is another line\nAnd another\n\n\nEnd\n"));
+    FREE_MEMORY(value);
+
+    EXPECT_STREQ(expected, value = CleanDoubleBackslashes("\\nThis is a test\n\nHere is another line\nAnd another\\n\n\\nEnd\\n"));
+    FREE_MEMORY(value);
+
+    EXPECT_STREQ(expected, value = CleanDoubleBackslashes("\nThis is a test\\n\\nHere is another line\\nAnd another\\n\n\\nEnd\n"));
+    FREE_MEMORY(value);
+
+    EXPECT_STREQ("\n\\Test\\123\n", value = CleanDoubleBackslashes("\n\\Test\\123\\n"));
```

Comment:
Only seeing single backslashes `\` and not double `\\` in your tests, should it also support double, as your function name suggests?

```
\\ This is a test \\
\\ Here is another line \\
\\ And another \\
```

Reply:
The double backslash this function addresses is "\n" being changed to "\\n" and with this EOL getting broken intro 2 meaningless characters. I'll add a comment to explain this for the function.

Reply:
ahh ok i see! Thanks for the explanation! Comments explaining this would be great 👍

Reply:
Better, changed the function name.